src/ebay-inline-notice/inline-notice.tsx

```diff
@@ -45,7 +45,7 @@ const EbayInlineNotice: FC<Props> = ({
         <div className={classNames(className, `inline-notice ${!isGeneral ? `inline-notice--${status}` : ``}`)}>
             {!isGeneral ? (
                 <span className="inline-notice__header">
-                    <EbayIcon name={`${status}-filled` as Icon} a11yText={ariaLabel} a11yVariant="label" />
+                    <EbayIcon name={`${status}FilledSmall` as Icon} a11yText={ariaLabel} a11yVariant="label" />
```

Comment:
Unfortunately, we are still using TypeScript 3.9.10 that's why we still need to assert type to Icon.

TypeScript 4.3 got [template string type improvements](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-3.html#template-string-type-improvements), meaning, we could remove the type assertion:

```tsx
<EbayIcon name={`${status}FilledSmall`} a11yText={ariaLabel} a11yVariant="label" />
```